modules/tfm/tfm/boards/common/crypto_keys.c

```diff
@@ -173,6 +176,7 @@ enum tfm_plat_err_t tfm_plat_builtin_key_get_lifetime_and_slot(mbedtls_svc_key_i
 
 enum tfm_plat_err_t tfm_plat_load_builtin_keys(void)
 {
+#if defined(CONFIG_HW_UNIQUE_KEY) || defined(TFM_PARTITION_INITIAL_ATTESTATION)
```

Comment:
Why did do you want to guard only the defines here?  If none of these are selected the function will just return success without any wanting that something is wrong. I suggest to move guard with the whole function with this and return an error if none of these is chosen.

Reply:
I want to return success if none of these are enabled, otherwise the system will not boot.
But I can move the ifdef guard around the entire thing, I need it only do prevent the warnings of  local variable defined but not used.

Reply:
I see, just for later it makes sense to ask them to add an ifdef  in the caller of this function.